orbmain/src/main/java/com/sun/corba/ee/impl/transport/DefaultSocketFactoryImpl.java

```diff
@@ -62,27 +62,6 @@ public ServerSocket createServerSocket(String type,
         return serverSocket;
     }
 
-    public Socket createSocket(String type, 
-                               InetSocketAddress inetSocketAddress)
-        throws IOException
-    {
-        SocketChannel socketChannel = null;
-        Socket socket = null;
-
-        if (orb.getORBData().connectionSocketType().equals(ORBConstants.SOCKETCHANNEL)) {
```

Comment:
Aha, here it is too ... type was unused, and it took the value from the orbdata ... why?